src/__test__/fetchCharacters.test.js

```diff
@@ -0,0 +1,21 @@
+import '@testing-library/jest-dom/extend-expect';
+import fetchData from '../__mocks__/https';
+import fetchCharacter from './fetchCharacterTesting';
+
+describe('test fetching characters', () => {
+  it('tests fetching of the alias', async () => {
+    await fetchCharacter().then((data) => {
+      expect(data[0].alias).toBe('Poop Hero');
+    });
+  });
+  it('tests fetching of the gender', async () => {
+    await fetchData().then((data) => {
+      expect(data[1].gender).toBe('Non-binary');
+    });
+  });
```

Comment:
- [ ] Some of your tests are failing, please kindly fix that. See image below

![test](https://user-images.githubusercontent.com/65852621/144339572-58cc87b2-f3ec-45e3-9941-06bf4cc20465.png)
.